reformat_gherkin/formatter.py

```diff
@@ -226,10 +222,12 @@ def __construct_contexts(self) -> ContextMap:
                 for row, line in zip(rows, lines):
                     contexts[row] = line
 
-        for node, next_node in zip_longest(nodes, nodes[1:], fillvalue=None):
             if isinstance(node, Comment):
-                # We want comments to have the same indentation level with the next line
-                contexts[node] = next_node
+                # We want comments to have the same indentation level as the nearest
+                # node that has an entry in the indent level map.
+                contexts[node] = comment_parent
+            elif type(node) in INDENT_LEVEL_MAP:
```

Comment:
This is the line that failed the Codacy check. It can be reformatted to `isinstance(node, tuple(INDENT_LEVEL_MAP.keys()))` to satisfy Codacy, but I think the original is clearer. What do you think?

Reply:
I agree that the original is clearer. You could try to add `# pylint: disable=unidiomatic-typecheck` at the end of line 229 to disable the check, since that rule is from PyLint.